certmanager/generate_cert.go

```diff
@@ -174,18 +189,40 @@ func genCertTemplate(options CertOptions) x509.Certificate {
 		KeyUsage:              x509.KeyUsageKeyEncipherment | x509.KeyUsageDigitalSignature,
 		ExtKeyUsage:           []x509.ExtKeyUsage{extKeyUsage},
 		BasicConstraintsValid: true,
-	}
-	hosts := strings.Split(options.Host, ",")
```

Comment:
shall we keep these options (DNS and IPaddress)? we might need that for testing when we work on enduser to TLS-secured server